contents/careers/full-stack-software-engineer.md

```diff
@@ -4,6 +4,74 @@ sidebar: Careers
\ No newline at end of file
+PostHog exists to increase the number of successful products in the world.
+
+## Engineering has been everything to us so far
+
+[James](https://twitter.com/james406) and [Tim](https://twitter.com/timgl) founded PostHog in January. They had a bunch of features they wanted to build to understand their own user behavior - not sharing user data to a third party service, autocapturing events, and getting access to underlying data. Things an engineer would care about.
+
+It wasn't until they met [Dalton Caldwell](https://twitter.com/daltonc?lang=en) for a chat that it all clicked - *open source* would be the strategy, everything else would follow.
+
+This means PostHog is a product-led company. We're building a big open source community around a great, totally free, product.
+
+The end result? Releasing new features regularly has been the *only* thing we've had to do to get thousands of deployments. We've raised millions of dollars from the world's best investors but we're still a small, lean team.
+
+PostHog helps improve products built by teams in everything from huge airlines and banks, to indie gamers making it more fun to protect earth from aliens and even underwear startups working on their retention!
+
+## The release schedule is wild
+
+We're up against billion dollar companies, and we're catching up. We will overtake them by being faster.
+
+[Comments like this](https://news.ycombinator.com/item?id=22378393) are what we live and die by.
+
+We keep cycles very short. There is a roadmap session every two weeks - each developer suggests what they want to build to have the greatest impact, then they do just that. You choose what you focus on, and you are responsible for tests, user feedback and tracking.
+
+## The people here are really, really smart
+
+Every day we feel lucky to have them here!
+
+We're all remote, and we raised enough to pay top of market. Combine those things with working on open source, and you have a recipe for hiring the best engineering talent in the world.
+
+You'd be working with a combination of former CTOs and YC founders all turned developers, alongside some of the best developers from the world's largest tech companies who have the experience to help us handle scalability.
+
+We take bets on people with less experience too - we are as willing to hire an unproven genius straight out of school as we are a seasoned veteran.
+
+Our team is tiny but we live in 10 different countries. This [diverse group](/handbook/company/diversity) looks [pretty weird](/handbook/company/team) on paper, but it's magical. Apart from the fact [most of them think pineapple belongs on pizza](https://twitter.com/PostHogHQ/status/1319583079648923648). It doesn't.
+
+## Get in early
+
+Would you have rather been in the first 20 or the last 20 people to join Google?
```

Comment:
comparison makes sense and we do want to give this feeling but I think the direct reference to Google is cheesy